src/components/wizard/wizard.component.ts

```diff
@@ -150,13 +150,13 @@ export class WizardComponent implements OnInit, OnDestroy {
     private _step: number = 0;
     protected _onDestroy = new Subject<void>();
 
-    constructor(protected _wizardService: WizardService) {
+    constructor(protected _wizardService: WizardService<WizardStepComponent | MarqueeWizardStepComponent>) {
```

Comment:
Remove `MarqueeWizardStepComponent` from this class. This class is the base class and should not know about any other uses of it. Otherwise this list could have to grow anytime we added a different wizard implementation. There should be no references to anything marquee wizard related in this class.